app/(auth)/reset-password.tsx

```
@@ -0,0 +1,221 @@
+      if (result.status === 'complete') {
+        // Password reset successful - sign in the user
+        await setActive({ session: result.createdSessionId });
+        toast.showToast('Password reset successful!', 'success');
+        // Navigation will be handled by layout component based on auth state
+      } else {
+        Alert.alert('Error', 'Could not reset password. Please try again.');
+      }
+    } catch (error: any) {
+      console.error('Password reset error:', error);
+      
+      if (error.errors && error.errors[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)